llvm/lib/Analysis/LazyValueInfo.cpp

```diff
@@ -1159,6 +1159,28 @@ getRangeViaSLT(CmpInst::Predicate Pred, APInt RHS,
   return std::nullopt;
 }
 
+/// Get value range for a "ctpop(Val) Pred RHS" condition.
+static ValueLatticeElement getValueFromICmpCtpop(ICmpInst::Predicate Pred,
+                                                 Value *RHS) {
+  unsigned BitWidth = RHS->getType()->getPrimitiveSizeInBits();
+
+  auto RHSConst = dyn_cast<ConstantInt>(RHS);
+  if (!RHSConst)
+    return ValueLatticeElement::getOverdefined();
+
+  auto &RHSVal = RHSConst->getValue();
```

Comment:
```suggestion
  const APInt &RHSVal = RHSConst->getValue();
```
Or just drop this variable.